src/test/resources/features/createCampaign.feature

```diff
@@ -6,6 +6,7 @@ Feature: Create Campaign
     When I create a new campaign with required fields
       | Campaign Name           | New Campaign |
     Then I verify Campaign created with requirement fields
```

Comment:
Could be 'I verify that the Campaign is created with the given required fields.

Reply:
good suggestion